<?xml version="1.0"?>
<launch>
	<arg name="_name" default="adrianohrl"/>
	<arg name="_mic_name" default="alsa_input.pci-0000_00_1b.0.analog-stereo"/>
	<group ns="$(arg _name)">
		<node name="system_user_interface_node" pkg="mrta_vc" type="system_user_interface_node" output="screen">
		</node>
		<node name="task_builder_node" pkg="mrta_vc" type="task_builder_node" output="screen">
		</node>
		<node name="speech_analyser_node" pkg="mrta_vc" type="speech_analyser_node" output="screen">
		</node>
		<node name="speech_recognizer_node" pkg="pocketsphinx" type="recognizer.py" output="screen">
			<param name="mic_name" value="$(arg _mic_name)"/>
			<param name="lm" value="$(find mrta_vc)/dic/robocup.lm"/>
			<param name="dict" value="$(find mrta_vc)/dic/robocup.dic"/>
		</node>
	</group>
</launch>
